src/common/util/save/save.h

```diff
@@ -22,7 +22,7 @@
 
 #include "common/universe.h"
 
-namespace cqsp::common::save {
+namespace cqsp::common::game {
```

Comment:
Why rename this to game? It should be save because I expect save games to be it's own relatively big thing.